doc/releases/release-notes-4.0.rst

```diff
@@ -266,6 +319,18 @@ Boards & SoC Support
 
   * Added Thingy53 forwarding of network core pins to network core for SPI peripheral (disabled
     by default) including pin mappings.
+  * Added uart, flexio pwm, flexio spi, watchdog, flash, rtc, i2c, lpspi, edma, gpio, acmp, adc and lptmr support
+    to NXP ``frdm_ke17z`` and ``frdm_ke17z512``
+  * Enabled support for MCUmgr on NXP boards
+  * Added documentation for SEGGER SystemView support  for all NXP boards with an SoC supporting ECC
+  * Enabled MCUboot, FlexCAN, LPI2C, VREF, LPADC and timers (TPM, LPTMR, counter, watchdog) on NXP ``frdm_mcxw71``
```

Comment:
the board itself was also new

Reply:
can we clarify somewhere that this board was added new or is that already somewhere else

Reply:
nevermind, looks like it's somewhere else